app/redux/utils/trackingMiddleware.js

```diff
@@ -25,7 +25,8 @@ const trackMetricMap = {
   SETUP_DATA_STORAGE_SUCCESS: 'Created Profile',
   UPDATE_PATIENT_SUCCESS: 'Updated Profile',
   UPDATE_USER_SUCCESS: 'Updated Account',
-  LOGOUT_REQUEST: 'Logged Out'
+  LOGOUT_REQUEST: 'Logged Out',
+  VERIFY_CUSTODIAL_SUCCESS: 'VCA Home Verification - Verified'
```

Comment:
Hmm, for testing these metrics (since that's on my mind), we'd need to add our metrics middleware to the config of the redux-mock-store and add the metrics to the expectations, yes? I think maybe we should try that.